go/cmd/berty/main.go

```diff
@@ -144,12 +140,19 @@ func main() {
 				remoteAddr = *miniClientDemoRemoteAddr
 			}
 
+			l := zap.NewNop()
+			if *globalLogToFile != "" {
+				l = logger
+			}
+
 			mini.Main(&mini.Opts{
-				RemoteAddr:      remoteAddr,
-				GroupInvitation: *miniClientDemoGroup,
-				Port:            *miniClientDemoPort,
-				RootDS:          rootDS,
-				Logger:          logger,
+				RemoteAddr:            remoteAddr,
+				GroupInvitation:       *miniClientDemoGroup,
+				Port:                  *miniClientDemoPort,
+				RootDS:                rootDS,
+				Logger:                l,
+				Bootstrap:             DefaultBootstrap,
```

Comment:
why not using RendezVousServer here too?

Reply:
It's because im not using the rdvp the same way in **mini** and daemon, I don't want to start mini until im fully connected with the rdvp,i gonna change the name and add flags to be more clear

Reply:
:+1: